tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -234,6 +235,7 @@ export function DataLoaderBehavior<Item, Data>(
               ).includes(LoadState.LOADING);
               if (!isDataFetchPending) {
                 this.dataLoading = false;
+                this.dataLoadedAtLeastOnce = true;
```

Comment:
iiuc, L238 is called in cases when the request was cancelled and there was no followup request, which is not desired.  Shouldn't this go inside the `if (!result.cancelled)` block?